iroh-relay/src/server/client.rs

```diff
@@ -651,108 +648,12 @@ mod tests {
                 packet: disco_data.clone().into(),
             })
             .await?;
-        let msg = server_channel_r.recv().await.unwrap();
-        match msg {
-            actor::Message::SendDiscoPacket {
-                dst: got_target,
-                src: got_src,
-                data: got_data,
-            } => {
-                assert_eq!(target, got_target);
-                assert_eq!(key, got_src);
-                assert_eq!(&disco_data[..], &got_data);
-            }
-            m => {
-                bail!("expected ServerMessage::SendDiscoPacket, got {m:?}");
-            }
-        }
 
         done.cancel();
         handle.await??;
         Ok(())
     }
 
-    #[tokio::test]
```

Comment:
deleting tests is clearly cheating to get your line diffs as net-removing lines :stuck_out_tongue:

Reply:
I also fixed some tests ? 😅